### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by TomPed**
We want a more standard style for the Codecov example `READMD.md`s. I've added and moved around a few things. Anyone please feel free to review the changes.

### CODE REVIEW

**Comment by stevepeak** [comment]
Ruby is different. You must use the Codecov Ruby gem: https://github.com/codecov/codecov-ruby

**Comment by TomPed** [comment]
Updated private repo section.

**Comment by brntbeer** [comment]
@TomPed can you update the OP (original post) here with what the goals / objectives of this PR are? I just wanna help out in any way i can!

**Comment by TomPed** [comment]
Updated 😄.

File: README.md [hunk 1]
```diff
@@ -1,12 +1,16 @@
-[Codecov][1] Ruby Example
-=========================
+# [Codecov][1] Ruby Example
+## Guide
+### Travis Setup
```
Comment (brntbeer) [inline README.md:3]: @TomPed To make this a bit more agnostic, what if we just made this an `##` and title it "CI setup" and add a  descriptive line of text mentioning that this is for travis CI but that most modern CIs with a `.yml` file should work appropriately. potentially linking to additional popular CI services like CircleCI and Jenkins.

additionally, i would removed the line "guide" above it, i think? It should be implicitly understood this is a guide from reading it, in my opinion

File: README.md [hunk 2]
```diff
@@ -1,12 +1,16 @@
...
+lang: ruby
 
-## See this repos [Coverage Reports][4]
+script: bundle exec rake
 
-
-## Usage
+rvm: 2.1
+```
+### Produce Coverage Reports
```
Comment (brntbeer) [inline README.md:13]: IMO this section is what confuses me the most between this repo and https://github.com/codecov/codecov-ruby. 

@TomPed what are your thoughts on changing this title to be "How this repo produces coverage reports" ?

**Review by brntbeer — CHANGES_REQUESTED** [review]
@TomPed overall i love the change this readme is taking to be more similar to other example repositories.

I think there's some strength in the README showing how to use the ruby library itself, but it may also benefit from a challenge to the reader to do something like "Fork this repo and achieve 100%!" by having them add the test themselves AND get exposed to seeing the integration in action! Just a thought though, I'm happy to do that as a follow up PR myself 😄 :octocat:

**Comment by thomasrockhu** [comment]
Closing in favor of https://github.com/codecov/example-ruby/pull/10

**Pull request closed without merging**